### ISSUE TITLE
Error when using 'spawn' as start method

### ISSUE THREAD

**Issue by yhcharles** [uqfoundation/multiprocess#145]
```python
import multiprocess as mp

def f(_):
    print('hi')

mp.set_start_method('spawn')
mp.Pool(3).map(f, range(3))
```

results error:
```python
python3.11/site-packages/multiprocess/util.py:452, in spawnv_passfds(path, args, passfds)
    450 errpipe_read, errpipe_write = os.pipe()
    451 try:
--> 452     return _posixsubprocess.fork_exec(
    453         args, [os.fsencode(path)], True, passfds, None, None,
    454         -1, -1, -1, -1, -1, -1, errpipe_read, errpipe_write,
    455         False, False, None, None, None, -1, None)
    456 finally:
    457     os.close(errpipe_read)

TypeError: fork_exec() takes exactly 23 arguments (21 given)
```

**Comment by mmckerns** [comment]
I'm unable to reproduce the issue you are seeing.
```
Python 3.11.3 (main, Apr  8 2023, 01:12:57) [Clang 10.0.1 (clang-1001.0.46.4)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import multiprocess as mp
>>> 
>>> def f(_):
...     print('hi')
... 
>>> mp.set_start_method('spawn')
>>> mp.Pool(3).map(f, range(3))
hi
hi
hi
[None, None, None]
>>> mp.__version__
'0.70.15.dev0'
>>> 
```

What's your version of python (3.11.x), x=?
What's your version of multiprocess?

Maybe a version update might be the solution for you.

**Comment by yhcharles** [comment]
My versions:
```
import multiprocess as mp
print(mp.__version__)
import sys
print(sys.version)

0.70.14
3.11.3 (main, Apr 19 2023, 23:54:32) [GCC 11.2.0]
```

From the original exception, it seems there is some mismatch in the `_posixsubprocess.fork_exec()` arguments.

**Comment by mmckerns** [comment]
Right...I believe the issue is that `0.70.14` does not formally support `3.11`.  It was released days before `3.11.0` was released, so it's configured for the last release candidate of `3.11.0`.  A new release is imminent, but until then, you'll need to use `0.70.15.dev0` for now if you want to use `3.11.3`.  So short answer it that this is already fixed, and will be in the next release.  If there's more to discuss, then feel free to reopen this.
